src/org/exist/collections/Collection.java

```diff
@@ -804,28 +816,6 @@ public DocumentImpl getDocumentWithLock(final DBBroker broker, final XmldbURI ur
         }
     }
 
-    public DocumentImpl getDocumentNoLock(final DBBroker broker, final String rawPath) throws PermissionDeniedException {
```

Comment:
I could have marked it deprecated... but it is unused in the eXist codebase, so it seemed more sensible to remove it.